src/RNCamera.js

```diff
@@ -465,6 +467,12 @@ export default class Camera extends React.Component<PropsType, StateType> {
     }
   };
 
+  _onStatusChange = () => {
+    if (this.props.onStatusChange) {
+      this.props.onStatusChange({ status: this.getStatus() });
```

Comment:
can you please also add `recordAudioPermissionStatus : this.state.recordAudioPermissionStatus`?